setup.py

```diff
@@ -19,27 +19,32 @@
 from basesetup import (write_version_py, build_ext,
                        StaticLibrary, CompilerDetection)
 
-try:
-    import numpy
-    import Cython
-    if Cython.__version__ < '0.28':
-        raise ImportError
-    from Cython.Build import cythonize
-except ImportError:
-    print('-'*80, file=sys.stderr)
-    print('''Error: building mdtraj requires numpy and cython>=0.19
-
-Try running the command ``pip install numpy cython`` or
```

Comment:
do we need to take these messages out? i imagine some people might still be using an older pip that doesn't support pep517, and maybe these messages would help them (maybe we should add a note here about upgrading pip instead of `conda install`ing numpy and cython.

Reply:
Some sort of message could be helpful, but perhaps it belongs in the installation docs. I would reserve messages in the actual installation scripts to assume you know what you want and help you fix them rather than telling you to do it another way. This is what led to my confusion in the first place, since I was providing Cython in my requirements and the error was still being raised. Until I saw the way the error was implemented I thought it was a problem with my `setup.py`. 
However, the conditional logic for the raising of these messages needs to be changed, because this will always fail when you try to `setup.py install` with cython in the build dependencies. Maybe there is a flag for pep517 or a pip version we can check for.